ui/app/AppLayouts/Chat/popups/community/CreateChannelPopup.qml

```diff
@@ -40,6 +40,14 @@ StatusDialog {
     signal editCommunityChannel(string chName, string chDescription, string chEmoji, string chColor, string chCategoryId)
     signal deleteCommunityChannel()
 
+    function openEmojiPopup(leftSide = false) {
```

Comment:
Does it make sense to make it private if it is only used internally?

Reply:
done